hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -70,7 +70,8 @@ public static SnapshotInfo getSnapshotInfo(final OzoneManager ozoneManager,
       throw e;
     }
     if (snapshotInfo == null) {
-      throw new OMException(KEY_NOT_FOUND);
+      throw new OMException("Snapshot '" + snapshotKey + "' is not found.",
```

Comment:
nit: Should it be `snapshotKey` or `snapshotName`? I think `snapshotName` should be better.

Reply:
@hemantk-12 In this case it only has `dbKey` (e.g. result of `SnapshotInfo.getTableKey(volumeName, bucketName, snapshotName)`)

Reply:
You are right. But I'm just thinking from client perspective `snapshotName` would be better which can be extracted from the `snapshotKey` I guess. At the same time it is not a big deal to return `snapshotKey` in exception message.